frontend/app/src/stores/xframe.ts

```diff
@@ -24,6 +24,8 @@ const FRAME_ANCESTORS = [
     "https://spyzr-gqaaa-aaaan-qd66q-cai.icp0.io", // vaultbet
     "https://bxnm7-oqaaa-aaaag-albpq-cai.icp0.io", // ZKP
     "https://panoramablock.com", // Panorama Block Live
+    "https://zdgud-kqaaa-aaaal-ajn4q-cai.icp0.io", // tendyzone test
+    "dragonwizards.club/lair.html" // dragon paladin wizard web2 test
```

Comment:
this one is differently wrong, but again needs to be just the origin (including the protocol) i.e. `https://dragonwizards.club`